iree/base/CMakeLists.txt

```diff
@@ -266,8 +266,7 @@ if(${IREE_ENABLE_RUNTIME_TRACING})
     DEPS
       ::core_headers
     DEFINES
-      # TODO(#2114): Change the mode to 2.
-      "IREE_TRACING_MODE=1"
+      "IREE_TRACING_MODE=2"
```

Comment:
Yay!

Reply:
It may still not work on linux (what #2114 was referencing), but the issue was from almost a year ago and we've bumped tracy versions since then. If there are still issues on Linux then someone will need to do a bit of sleuthing to figure out what's wrong as there is likely a legit issue only showing up there that is worth fixing :)